### PR TITLE
Fix Fuchsia build

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

**Comment by tvolkert** [comment]
@abarth

**Comment by pylaligand** [comment]
LGTM

File: packages/flutter_tools/bin/fuchsia_builder.dart [hunk 1]
```diff
@@ -36,6 +37,7 @@ Future<Null> main(List<String> args) async {
   executableContext.setVariable(Logger, new StdoutLogger());
   executableContext.runInZone(() {
     // Initialize the context with some defaults.
+    context.putIfAbsent(ProcessManager, () => new ProcessManager());
```
Comment (abarth) [inline packages/flutter_tools/bin/fuchsia_builder.dart]: Can you add a comment to wherever else this list exists that cross-references this copy?  Even better, can we share the code so we don't need to keep these two lists in sync?

Comment (tvolkert) [inline packages/flutter_tools/bin/fuchsia_builder.dart]: The lists aren't the same (Fuchsia is a subset of the main executable).  Added note.

**Pull request merged**